---
title: Resources
---
<div class="container">

  <div class="d-flex flex-column">

    <!-- Section -->
    <div>
      <div class="publication" id="YEP">
        <h4>Yeast Epigenome Project</h4>
        <hr>
        <small>
          Matthew J. Rossi, Prashant K. Kuntala, William K.M. Lai, Naomi Yamada, Nitika Badjatia,Chitvan Mittal, Guray Kuzu, Kylie Bocklund, Nina P. Farrell, Thomas R. Blanda, Joshua D. Mairose, Ann V. Basting, Katelyn S. Mistretta, David J. Rocco, Emily S. Perkinson, Gretta D.Kellogg, Shaun Mahony, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>Proteins are architected along eukaryotic chromosomes to maintain chromosome integrity and
        regulate thousands of genes. There exists no holistic single-base resolution map of their
        structural organization that reflects on genome function. Here we use ChIP-exo to define this
        structure in the yeast Saccharomyces. We identified 21 meta-assemblages consisting of >400
        proteins related to replication origins, centromeres, subtelomeric regions, transposons, and RNA
        polymerase (Pol) I, II, and III transcription units. Most Pol II promoters lacked a regulatory
        region by design. These essentially constitutive promoters comprised a short nucleosome-free
        region adjacent to a +1 nucleosome, which together bound TFIID to formed a pre-initiation
        complex (PIC). Positioned insulators protected core promoters from upstream events. Only 1/5th
        of all promoters were architected for inducible regulation, wherein combinations of 78 sequence24 specific transcription factors bound upstream to create a nucleosome-depletable regulatory
        region. We describe their structural interactions with the genome and cognate cofactors,
        including nucleosomal and transcriptional regulators RPD3-L, SAGA, NuA4, Tup1, Mediator,
        and SWI/SNF. Together their assemblages are linked to PIC assembly involving primarily TBP
        rather than TFIID to achieve a defined integrated network of regulated transcription.
        </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="http://yeastepigenome.org" title="project website"> Resource <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://pubmed.ncbi.nlm.nih.gov/33692541/" title="pubmed link"> PubMed <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://www.nature.com/articles/s41586-021-03314-8" title="journal link"> Nature <i class="fas fa-external-link-alt"></i></a>
        </div>
      </div>

    </div>

    <!-- Section -->
    <div>
      <div class="publication" id="PCRP">
        <h4>Protein Capture Reagents Program Validation</h4>
        <hr>
        <small>
          William K. M. Lai, Luca Mariani, Gerson Rothschild, Edwin R. Smith, Bryan J. Venters, Thomas R. Blanda, Prashant K. Kuntala, Kylie Bocklund, Joshua Mairose, Sarah N Dweikat, Katelyn Mistretta, Matthew J. Rossi, Daniela James, James T. Anderson, Sabrina K. Phanor, Wanwei Zhang, Avani P. Shaw, Katherine Novitzky, Eileen McAnarney, Michael-C. Keogh, Ali Shilatifard, Uttiya Basu, Martha L. Bulyk, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>
          Antibodies offer a powerful means to interrogate specific proteins in a complex milieu, and where epitope tagging is impractical. However, antibody availability and reliability are problematic. The Protein Capture Reagents Program (PCRP) generated over a thousand renewable monoclonal antibodies against human-presumptive chromatin proteins in an effort to improve reliability. However, these reagents have not been widely field-tested. We therefore screened their ability in a variety of assays. 887 unique antibodies against 681 unique chromatin proteins, of which 605 are putative sequence-specific transcription factors (TFs), were assayed by ChIP-exo. Subsets were further tested in ChIP-seq, CUT&RUN, STORM super-resolution microscopy, immunoblots, and protein binding microarray (PBM) experiments. At least 6% of the tested antibodies were validated for use in ChIP-based assays by the most stringent of our criteria. An additional 34% produced data suggesting they warranted further testing for clearer validation. We demonstrate and discuss the metrics and limitations to antibody validation in chromatin-based assays.
        </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="http://odin.cac.cornell.edu/pcrp/" title="project website"> Resource <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://pubmed.ncbi.nlm.nih.gov/34426512/" title="pubmed link"> PubMed <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://genome.cshlp.org/content/31/9/1663.long" title="journal link"> Genome Research <i class="fas fa-external-link-alt"></i></a>
        </div>
      </div>

    </div>

    <!-- Section -->
    <div>
      <div class="publication" id="PEGR">
        <h4>PEGR: a management platform for ChIP-based next generationsequencing pipelines</h4>
        <hr>
        <small>
          Danying Shao, Gretta D.Kellogg, William K.M. Lai, Shaun Mahony, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>
          There has been a rapid development in genome sequencing, in-cluding high-throughput next generation sequencing (NGS) tech-nologies, automation in biological experiments, new bioinformaticstools and utilization of high-performance computing and cloudcomputing. ChIP-based NGS technologies, e.g. ChIP-seq and ChIP-exo, are widely used to detect the binding sites of DNA-interactingproteins in the genome and help us to have a deeper mechanisticunderstanding of genomic regulation. As sequencing data is gener-ated at an unprecedented pace from the ChIP-based NGS pipelines,there is an urgent need for a metadata management system. Tomeet this need, we developed the Platform for Eukaryotic GenomicRegulation (PEGR), a web service platform that logs metadata forsamples and sequencing experiments, manages the data processingworkflows, and provides reporting and visualization. PEGR linkstogether people, samples, protocols, DNA sequencers and bioinfor-matics computation. With the help of PEGR, scientists can have amore integrated understanding of the sequencing data and betterunderstand the scientific mechanisms of genomic regulation. In thispaper, we present the architecture and the major functionalities ofPEGR. We also share our experience in developing this applicationand discuss the future directions.
        </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="https://github.com/seqcode/pegr" title="project website"> Github <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://pubmed.ncbi.nlm.nih.gov/35440038/" title="pubmed link"> PubMed <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02671-5" title="journal link"> Genome Biology <i class="fas fa-external-link-alt"></i></a>
        </div>
      </div>

    </div>

  </div>
</div>
