---
title: Resources
---
<div class="container">

  <div class="d-flex flex-column">

    <!-- Section -->
    <div>
      <div class="publication" id="YEP">
        <h4>Yeast Epigenome Project</h4>
        <hr>
        <small>
          Matthew J. Rossi, Prashant K. Kuntala, William K.M. Lai, Naomi Yamada, Nitika Badjatia,Chitvan Mittal, Guray Kuzu, Kylie Bocklund, Nina P. Farrell, Thomas R. Blanda, Joshua D. Mairose, Ann V. Basting, Katelyn S. Mistretta, David J. Rocco, Emily S. Perkinson, Gretta D.Kellogg, Shaun Mahony, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>
The genome-wide architecture of chromatin-associated proteins that maintains chromosome integrity and gene regulation is ill defined. Here we use chromatin immunoprecipitation, exonuclease digestion and DNA sequencing (ChIP–exo/seq) to define this architecture in Saccharomyces cerevisiae. We identify 21 meta-assemblages consisting of roughly 400 different proteins that are related to DNA replication, centromeres, subtelomeres, transposons, and transcription by RNA polymerase (Pol) I, II and III. Replication proteins engulf a nucleosome, centromeres lack a nucleosome, and repressive proteins encompass three nucleosomes at subtelomeric X-elements. We find that most promoters associated with Pol II evolved to lack a regulatory region, having only a core promoter. These constitutive promoters comprise a short nucleosome-free region (NFR) adjacent to a +1 nucleosome, which together bind the transcription-initiation factor TFIID to form a preinitiation complex (PIC). Positioned insulators protect core promoters from upstream events. A small fraction of promoters evolved an architecture for inducibility, whereby sequence-specific transcription factors (ssTFs) create a nucleosome-depleted region (NDR) that is distinct from an NFR. We describe structural interactions among ssTFs, their cognate cofactors and the genome. These interactions include the nucleosomal and transcriptional regulators RPD3-L, SAGA, NuA4, Tup1, Mediator and SWI–SNF. Surprisingly, we do not detect interactions between ssTFs and TFIID, suggesting that such interactions do not stably occur. Our model for gene induction involves ssTFs, cofactors and general factors such as TBP and TFIIB, but not TFIID. By contrast, constitutive transcription involves TFIID but not ssTFs and cofactors. From this, we define a highly integrated network of gene regulation by ssTFs.
                </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="http://yeastepigenome.org" title="project website" >Web <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://www.nature.com/articles/s41586-021-03314-8" title="pubmed link"> (published in Nature)</a>
        </div>
      </div>

    </div>

    <!-- Section -->
    <div>
      <div class="publication" id="PCRP">
        <h4>Protein Capture Reagents Program Validation</h4>
        <hr>
        <small>
          William K. M. Lai, Luca Mariani, Gerson Rothschild, Edwin R. Smith, Bryan J. Venters, Thomas R. Blanda, Prashant K. Kuntala, Kylie Bocklund, Joshua Mairose, Sarah N Dweikat, Katelyn Mistretta, Matthew J. Rossi, Daniela James, James T. Anderson, Sabrina K. Phanor, Wanwei Zhang, Avani P. Shaw, Katherine Novitzky, Eileen McAnarney, Michael-C. Keogh, Ali Shilatifard, Uttiya Basu, Martha L. Bulyk, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>
          Antibodies offer a powerful means to interrogate specific proteins in a complex milieu, and where epitope tagging is impractical. However, antibody availability and reliability are problematic. The Protein Capture Reagents Program (PCRP) generated over a thousand renewable monoclonal antibodies against human-presumptive chromatin proteins in an effort to improve reliability. However, these reagents have not been widely field-tested. We therefore screened their ability in a variety of assays. 887 unique antibodies against 681 unique chromatin proteins, of which 605 are putative sequence-specific transcription factors (TFs), were assayed by ChIP-exo. Subsets were further tested in ChIP-seq, CUT&RUN, STORM super-resolution microscopy, immunoblots, and protein binding microarray (PBM) experiments. At least 6% of the tested antibodies were validated for use in ChIP-based assays by the most stringent of our criteria. An additional 34% produced data suggesting they warranted further testing for clearer validation. We demonstrate and discuss the metrics and limitations to antibody validation in chromatin-based assays.
        </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="http://odin.cac.cornell.edu/pcrp/" title="project website" >Web <i class="fas fa-external-link-alt"></i></a>
          <a class="publinks" href="https://doi.org/10.1101/2020.06.08.140046" title="link"> BioRxiv <i class="fas fa-file-pdf"></i></a>
        </div>
      </div>

    </div>

    <!-- Section -->
    <div>
      <div class="publication" id="PEGR">
        <h4>PEGR: a management platform for ChIP-based next generationsequencing pipelines</h4>
        <hr>
        <small>
          Danying Shao, Gretta D.Kellogg, William K.M. Lai, Shaun Mahony, B. Franklin Pugh
        </small>
        <hr>
        <h5>Abstract</h5>
        <p>
          There has been a rapid development in genome sequencing, in-cluding high-throughput next generation sequencing (NGS) tech-nologies, automation in biological experiments, new bioinformaticstools and utilization of high-performance computing and cloudcomputing. ChIP-based NGS technologies, e.g. ChIP-seq and ChIP-exo, are widely used to detect the binding sites of DNA-interactingproteins in the genome and help us to have a deeper mechanisticunderstanding of genomic regulation. As sequencing data is gener-ated at an unprecedented pace from the ChIP-based NGS pipelines,there is an urgent need for a metadata management system. Tomeet this need, we developed the Platform for Eukaryotic GenomicRegulation (PEGR), a web service platform that logs metadata forsamples and sequencing experiments, manages the data processingworkflows, and provides reporting and visualization. PEGR linkstogether people, samples, protocols, DNA sequencers and bioinfor-matics computation. With the help of PEGR, scientists can have amore integrated understanding of the sequencing data and betterunderstand the scientific mechanisms of genomic regulation. In thispaper, we present the architecture and the major functionalities ofPEGR. We also share our experience in developing this applicationand discuss the future directions.
        </p>
        <!-- image grid or slideshow of screenshots -->
        <hr>
        <div class="d-flex justify-content-end">
          <a class="publinks" href="http://pegr.org" title="project website" >Web <i class="fas fa-external-link-alt"></i></a>
          <!-- <a class="publinks" href="#" title="link"> <i class="fas fa-file-pdf"></i></a> -->
        </div>
      </div>

    </div>

  </div>
</div>
